src/Uno.Toolkit.UI/Controls/AutoLayout/AutoLayout.cs

```diff
@@ -41,6 +43,11 @@ protected override void OnApplyTemplate()
 			UpdateAutoLayout();
 		}
 
+		private void OnLoaded(object sender, RoutedEventArgs e)
+		{
+			UpdateAutoLayout();
+		}
```

Comment:
```suggestion
		private static void OnLoaded(object sender, RoutedEventArgs e)
		{
			(sender as AutoLayout)?.UpdateAutoLayout();
		}
```

This will prevent any possible memory leak (theoretically your code is right, but `GC` might be touchy especially on iOS), and is also _probably_ slightly more performant since it will create only one instance of delegate instead of creating one per instance of the `AutoLayout`.

Reply:
great thanks a lot @dr1rrb !